openid-4-verifiable-presentations-1_0.md

```diff
@@ -1128,6 +1128,14 @@ If the JWT is only a JWE, the following processing rules MUST be followed:
 - `iss`, `exp` and `aud` MUST be omitted in the JWT Claims Set of the JWE, and the processing rules as per [@!JARM] Section 2.4 related to these claims do not apply.
 - The processing rules as per [@!JARM] Section 2.4 related to JWS processing MUST be ignored.
 
+Whenever the response is encrypted, the following additional processing directives apply to bind the Verifier's identifier and its respective transaction with the End-User into the JWE:
+
+- The `apu` (Agreement PartyUInfo) header parameter MUST be set to the value of the `nonce` from the Authorization Request.
+- The `apv` (Agreement PartyVInfo) header parameter MUST be set to the value of the `client_id` of the Verifier.
```

Comment:
Should we use only `apv` for this, i.e., client_id:nonce? Since it is the recipient info and nonce was provided by the recipient. This would allow us to put other things into `apu` if needed.